test/integration/params-validations-test.js

```diff
@@ -127,4 +127,24 @@ describe('params validations', () => {
       due_on: new Date('2012-10-09T23:39:01Z')
     })
   })
+
+  it('Date is passed in correct format for notifications (#716)', () => {
```

Comment:
@JuanCaicedo I added the `(#716)` to the test description so folks can read up on the discussion behind this test